controllers/funding/programme-detail.js

```diff
@@ -0,0 +1,33 @@
+const { get } = require('lodash');
+const contentApi = require('../../modules/content-api');
+const { renderNotFound } = require('../http-errors');
+
+module.exports = function(config) {
+    return function(req, res) {
+        contentApi
+            .getFundingProgramme({
+                locale: req.i18n.getLocale(),
+                slug: req.params.slug
+            })
+            .then(entry => {
+                const contentSectionsRaw = get(entry, 'contentSections', []);
+                const contentSections = contentSectionsRaw.map((section, idx) => {
+                    return {
+                        id: `section-${idx}`,
+                        title: section.title,
+                        content: section.body,
+                        active: idx === 0
+                    };
+                });
+
+                res.render(config.template, {
+                    title: entry.title,
+                    contentSections: contentSections
+                });
+            })
+            .catch(err => {
+                console.log(err);
```

Comment:
Should we send this to Sentry?

Reply:
Hmm, I left this console log in by mistake. Buy maybe still yeah.